Automattic/jetpack
# Jetpack: implement a temporary solution to register VideoPress block v6 in dotcom

This temporary (but still functional) change allows registering the new VideoPress block in Simple sites. We'll remove this code once the new Jetpack-on-dotcom release happens.

Fixes #

#### Changes proposed in this Pull Request:

* Jetpack: implement a temporary solution to register VideoPress block v6 in dotcom

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Test in simple sites
* Apply the patch
* Get sandboxed
* Confirm you see the block
